packages/cosmic-swingset/lib/ag-solo/vats/issuers.js

```diff
@@ -21,76 +20,227 @@ export const CENTRAL_ISSUER_NAME = 'MOE';
  * @property {Brand} [brand]
  * @property {Array<any>} [issuerArgs]
  * @property {CollateralConfig} [collateralConfig]
- * @property {string} pursePetname
- * @property {Bigish} mintValue
- * @property {Array<[Bigish, Bigish]>} [fakeTradesGivenCentral]
+ * @property {Array<[string, Bigish]>} [defaultPurses]
+ * @property {Array<[Bigish, Bigish]>} [tradesGivenCentral]
  */
 
+export const makeScaler = toDecimals => (n, fromDecimals = 0) => {
+  if (typeof n === 'bigint') {
```

Comment:
shouldn't this be checking that `fromDecimals` is 0?

Reply:
perhaps he method needs a comment

Reply:
Done.

Reply:
Actually, the separate branch is for bigint values specifically (for which `fromDecimals` doesn't matter).  The other logic handles the case where a `number` is specified, regardless of `fromDecimals` value.

I've added comments to that effect.